File: Dart/alg09.dart
/* ALG09 */
import 'dart:io';
import 'dart:math';
void main() {
  int X;
  int Y;
  X = int.parse(stdin.readLineSync()!);
  if (X >= 10)
    Y = pow(X, 2).toInt();
  else
    Y = pow(X, 3).toInt();
  print('$Y');
}

File: Scala/alg09.scala
/* ALG09 */
import scala.io.StdIn
import scala.math.pow
object Estudo extends App {
  var X: Int = 0
  var Y: Int = 0
  X = scala.io.StdIn.readInt()
  if (X >= 10)
    Y = pow(X, 2).toInt
  else
    Y = pow(X, 3).toInt
  println(Y)
}
